### PR TITLE
Use Blockchain base node

### PR DESCRIPTION
**Pull request by jhuntbach-bc**

### CODE REVIEW

File: config-dev.json [hunk 1]
```diff
@@ -146,7 +146,7 @@
       "nativeAsset": "BASEETH",
       "networkTicker": "BASEETH",
       "nodeUrls": [
-        "https://mainnet.base.org"
+        "https://api.blockchain.info/base/nodes/rpc/"
```
Comment (vlenoir-bc) [inline config-dev.json:149]: Don't we want: `https://api.dev.blockchain.info/base/nodes/rpc` here? 🤔 
It does not change much but I think it's exposed as well

Comment (jhuntbach-bc) [reply config-dev.json:149]: Uhh I'm not sure. The prod one definitely works, but I'm getting a 503 from staging.

```
curl -X POST https://api.staging.blockchain.info/base/nodes/rpc/  -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}'

<html>
<head><title>503 Service Temporarily Unavailable</title></head>
<body>
<center><h1>503 Service Temporarily Unavailable</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Comment (vlenoir-bc) [reply config-dev.json:149]: But dev does work: 

```
curl -X POST https://api.dev.blockchain.info/base/nodes/rpc  -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' 
{"jsonrpc":"2.0","id":1,"result":"0x12ae5d7"}
```

Comment (jhuntbach-bc) [reply config-dev.json]:
```suggestion
        "https://api.dev.blockchain.info/base/nodes/rpc/"
```

**Pull request merged**